code/modules/spells/spell_types/mind_transfer.dm

```diff
@@ -9,8 +9,7 @@
 	invocation_type = "whisper"
 	range = 1
 	cooldown_min = 200 //100 deciseconds reduction per rank
-	var/list/protected_roles = list("Wizard","Changeling","Cultist") //which roles are immune to the spell
-	var/paralysis_amount_caster = 20 //how much the caster is paralysed for after the spell
+	var/paralysis_amount_caster = 15 //how much the caster is paralysed for after the spell

```

Comment:
Please remember that second values for Stun, Weaken, and Paralyze are doubled; this is not 15 seconds, it is 30.